darwin/dataset/remote_dataset.py

```diff
@@ -658,9 +658,9 @@ def split(
 
         Parameters
         ----------
-        val_percentage : int
+        val_percentage : float
             Percentage of images used in the validation set
-        test_percentage : int
+        test_percentage : float
```

Comment:
If we do this (working with `float` instead of `int`) we will be breaking the Public API of several functions. 
The reason I changed everything to `int` its because the code inside was working with integers instead of floats. Not only that, the majority of functions used `int`, so I went with the change that was least disruptive.

I don't oppose the changes, but I would like @andreaazzini opinion, since this merits a 0.8.0 release IMO.

Reply:
Then let's change them, float is the correct value to use here and IMHO it will make more sense for all our users.

I don't understand why everytime we need to be scared if we improve our APIs

Reply:
Mmh I don't think I agree here guys. That's a type annotation, which as you know in Python is not going to change the way you use your API